### DISCUSSION TITLE
tusb.h

### DISCUSSION THREAD

**Discussion by tyustli** [hathach/tinyusb#2521] [General]
why tusb.h #include "class/hid/hid.h" without macro?

**Comment by tyustli** [comment]
It is a feature?

![image](https://github.com/hathach/tinyusb/assets/43946994/6982349a-e24a-42cf-bbd8-be22ea7ffaa4)

**Reply by tyustli** [reply]
@hathach

**Reply by hathach** [reply]
It should have a macro guard as other, I think I did a quick hack for reusing HID definittion when doing ble (HID over GATTS) on nrf52833 or something. Should be reverted.
